### PR TITLE
pg_config.h: NetBSD support

### PR DESCRIPTION
**Pull request by bsiegert**
This is the same as pganalyze/pg_query_go#72 except one level higher.

NetBSD has `strchrnul` too but with a slightly different prototype. Without this change, there is a compiler error for a conflicting declaration.

### CODE REVIEW

File: Makefile
Comment (lfittl) [inline Makefile:150]: We do actually also need to apply it to the generated pg_config.h file in the repo (we cache the result of the Makefile and only run extract_source as needed) -- you can just make the edit directly in the file, or run the `extract_source` locally to generate it (but that might be a bit more involved):

https://github.com/pganalyze/libpg_query/blob/15-latest/src/postgres/include/pg_config.h#L1035

**Review by lfittl — COMMENTED** [review]
Thanks for getting this change in here!

There is actually one more small detail (I wasn't fully clear in my initial ask I realize), once that's fixed we can merge it :)

File: Makefile
Comment (bsiegert) [reply Makefile:150]: Done. I did not manage to get `make extract_source` to work (neither on NetBSD nor on Debian) so this is a manual edit. PTAL :)

**Review by lfittl — APPROVED** [review]

**Comment by bsiegert** [comment]
Can someone merge this, because I cannot :)

**Comment by lfittl** [comment]
> Can someone merge this, because I cannot :)

Looks like there was a seemingly random build failure - will merge after thats resolved (hopefully on a re-run, that I started just now).

**Pull request merged**
